lib/iex/lib/iex/helpers.ex

```diff
@@ -560,19 +575,32 @@ defmodule IEx.Helpers do
 
   defp expand_home(other), do: other
 
-  defp ls_print(_, []) do
+  @doc """
+  Prints a list in table format
+
+  First argument must be a list of string to print.
+
+  Second argument is optional and can be an anonymous function that format table cell.
+  Example: 
+      print_function = fn(item, opts, offset) -> String.pad_trailing(item, offset) end
+
+  Third argument is optional and can be a keyword list of values that can be used in format function
+  """
+  @spec tp([String.t], function, Keyword.t) :: any
+  def tp(list, print_function \\ fn(item, _opts, offset) -> String.pad_trailing(item, offset) end, opts \\ [])
```

Comment:
If we remove the opts argument from the anonymous function, which does not seem to be necessary, then the default print function can be `&String.pad_trailing/2`.